releng/pom.xml

```diff
@@ -12,6 +12,12 @@
 		<root-dir>${basedir}/..</root-dir>
 	</properties>
 
+	<pluginRepositories>
```

Comment:
calling `./gradlew generateP2Build` will move these changes to the void again unfortunately

Reply:
https://github.com/TypeFox/gradle-plugins/blob/9742b36882f062d272917528c5f5ccfec3ee21a5/gradle-p2gen/src/main/java/io/typefox/gradle/p2gen/P2GenPlugin.xtend

Reply:
=> we would need to fix that there or get rid of the p2gen an maintain the stuff manually as we did in Xtext

Reply:
OK. I hadn't realized it was being generated (TODO add "this file is auto-generated" comment)

Reply:
For xtext did you just start with the generated file and remove the generateP2Build?

Reply:
yes.

Reply:
i am not sure if we can change the typefox plugin to add the repo and profile

Reply:
I have changed this to be manually generated and removed generateP2Build.

Reply:
ok. can you please  create a follow up task to discuss this with @spoenemann or @dhuebner  once they are back?